File: SourceCode/EPONS-Mobile/Bimbo/app/src/main/java/com/epons/bimbo/models/Patient.java
package com.epons.bimbo.models;

/**
 * Created by Barend.Erasmus on 4/12/2016.
 */
public class Patient {
    public String Firstname = "Barend";
    public String Lastname;


    public String Fullname() {
        return Firstname + " " + Lastname;
    }
}

File: SourceCode/EPONS/EPONS.Teddy.Application/EntityViews/Patient.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EPONS.Teddy.Application.EntityViews
{
    public class Patient
    {
        public Guid Id { get; set; }
        [DisplayName("First Name")]
        public string Firstname { get; set; }
        [DisplayName("Last Name")]
        public string Lastname { get; set; }
        [DisplayName("Date of Birth")]
        public DateTime DateOfBirth { get; set; }
        public string Gender { get; set; }
        public string Race { get; set; }
        [DisplayName("Medical Scheme")]
        public string MedicalScheme { get; set; }
        public string Facilities { get; set; }
    }
}
